occupi-backend/pkg/mail/email_format.go

```diff
@@ -115,10 +149,11 @@ func appendHeader(title string) string {
 }
 
 func appendFooter() string {
-	return `<div class="footer">
-				<img src="https://raw.githubusercontent.com/COS301-SE-2024/occupi/develop/presentation/Occupi/Occupi-black.png" alt="Business Banner" style="width:100%;">
-				<p>140 Lunnon Road, Hillcrest, Pretoria. PO Box 14679, Hatfield, 0028</p>
-			</div>
+	return `
```

Comment:
@Rethakgetse-Manaka did you verify that the styling of this footer looks alright? Because I noticed that you are using fixed width's like 600px which may cause issues on people who have monitors that are really wide

Reply:
Yeah I did test it to verify on the other one was very big and looked weird